Can't resolve 'file-selector'

After installing react-dropzone
npm install --save react-dropzone
I have the following error when trying to start the application.
./node_modules/react-dropzone/dist/es/index.js
Module not found: Can't resolve 'file-selector' in 'C:\src\app\node_modules\react-dropzone\dist\es'

The version is  "react-dropzone": "^11.2.0",

I still got **Module not found: Can't resolve 'file-selector'**. 
Add below to package.json resolved the issue.
 
"resolutions": {
    "file-selector": "^v0.2.2"
  }

--- a/package.json
+++ b/package.json
@@ -91,7 +91,7 @@
   },
   "dependencies": {
     "attr-accept": "^2.2.1",
-    "file-selector": "^0.1.12",
+    "file-selector": "^0.2.2",
     "prop-types": "^15.7.2"
   },
   "devDependencies": {

